src/development/add-to-app/ios/project-setup.md

```diff
@@ -478,3 +504,5 @@ You can now [add a Flutter screen][] to your existing application.
 [VS Code]: {{site.url}}/development/tools/vs-code
 [XCFrameworks]: {{site.apple-dev}}/documentation/xcode_release_notes/xcode_11_release_notes
 [Xcode installed]: {{site.url}}/get-started/install/macos#install-xcode
+[News Feed app]: https://github.com/flutter/put-flutter-to-work/tree/022208184ec2623af2d113d13d90e8e1ce722365
```

Comment:
do we want po provide 2 examples? (one with app delegate and the other without)

Reply:
I imagine that someone could adapt the example based on the inline code we provide, but I'm sure it wouldn't hurt to have more examples. I don't think it makes sense to prioritize creating a full example right now, but maybe we can revisit after some of the other things (extensions, companion app, etc) have been added? WDYT?

Reply:
sounds good